src/TorSharp/TorSharp.csproj

```diff
@@ -53,6 +53,8 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <IncludeSymbols>true</IncludeSymbols>
+    <AssemblyVersion>19.11.11.2249</AssemblyVersion>
```

Comment:
What do these `AssemblyVersion` and `FileVersion` changes do? Are they intended or necessary?

Reply:
I use an extension that rename automatically the version yy.mm.dd...
You can ignore this